README.md

```diff
@@ -1005,6 +1007,90 @@ expect(multiple).to allow_transition_to(:processing).on(:move)
 expect(multiple).to_not allow_transition_to(:sleeping).on(:move)
 ```
 
+#### Minitest
+
+AASM provides assertions and rspec-like expectations for [Minitest](https://github.com/blowmage/minitest-rails).
```

Comment:
I think `minitest` is provided by [seattlerb](https://github.com/seattlerb/minitest) (and Ryan would be really sad if we prefer the Rails wrapper gem).